styles.css

```diff
@@ -0,0 +1,1146 @@
+}
+
+.about-description {
+  margin: 2.5rem auto 0;
+  background-color: rgb(212, 209, 209);
+  padding: 6%;
+  border: 1px solid var(--border-lines-color);
+}
+
+.contact-text {
+  background-color: rgb(212, 209, 209);
+  text-align: center;
+  font-size: 0.8rem;
+  padding: 1.5rem 2rem;
+  color: var(--main-text-color);
+}
+
+.email-contact {
+  background-color: rgb(212, 209, 209);
+  font-size: 0.8rem;
+  text-decoration: underline;
+  text-align: center;
+}
+
+#logo-section {
+  background-color: #fff;
+  padding: 2rem 0;
+}
+
+.logo-heading,
+.logo-description {
+  text-align: center;
+  color: var(--main-text-color);
+  padding: 0.5rem 1rem;
+}
+
+.logo-heading {
+  margin: 0 auto;
+  max-width: 80%;
+}
+
+.logo-description {
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+}
+
+.logo-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 0.5px solid var(--border-lines-color);
+  width: 60%;
+  padding: 1rem 2rem;
+  margin: 2rem auto;
+}
+
+#about {
+  background-color: #d3d3d3;
+  padding-bottom: 1rem;
+}
+
+.about-logo-img {
+  width: 150px;
+  height: auto;
+}
+
+.about-logo-text {
+  font-size: 0.9rem;
+  font-weight: 900;
+  width: 190px;
+}
+
+.date-location {
+  font-size: 0.5rem;
+  margin: 0.3rem 0;
+  border-top: 2px solid var(--border-lines-color);
+  border-bottom: 2px solid var(--border-lines-color);
+  text-align: left;
+  color: var(--main-text-color);
+}
+
+/******* */
+
+#upcomming {
+  padding-top: 2rem;
+  text-align: center;
+}
+
+.upcomming-heading {
+  color: var(--main-text-color);
+}
+
+.upcomming-description {
+  color: var(--main-text-color);
+  padding: 1rem 0 2rem;
+  font-size: 0.9rem;
+  max-width: 80%;
+  margin: 0 auto;
+}
+
+.video-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-bottom: 6rem;
+  row-gap: 2rem;
+}
+
+.video-background-1 {
+  background: url("");
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 300px;
+  height: 200px;
+  position: relative;
+}
+
+.video-overlay-1 {
+  background-color: rgba(236, 83, 66, 0.58);
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.video-background-2 {
+  background: url("./images/2022-congress-img.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 300px;
+  height: 200px;
+  position: relative;
+}
+
+.video-overlay-2 {
+  background-color: rgba(236, 83, 66, 0.58);
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.video-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+.year,
+.past-year-description {
+  color: #fff;
+}
+
+.footer-about {
+  background-color: var(--main-text-color);
+}
+
+.footer-heading-about,
+.footer-description-about {
+  color: var(--program-text-color);
+}
+
+/* about page end */
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] You chose the min-width of the screen of `768px` which means when the screen is this size the desktop version design should be worked correctly but this is not the case for you. You should make sure your website is responsive. Kindly fix your design for the home and about page 😉 🙃 

![127 0 0 1_5500_](https://user-images.githubusercontent.com/74501523/212444872-e0ba5ca2-f373-4ff3-ba8c-119c014bfd28.png)